### PR TITLE
Site creation

### PR DESCRIPTION
**Pull request by ngilain**

### CODE REVIEW

File: en/hosting_guide/site_creation.rst [hunk 1]
```diff
@@ -0,0 +1,106 @@
+Website creation
+================
+
+Creating a new site with Open Orchestra requires three steps. Additionnally, you can add some more
```
Comment (itkg-rboessel) [inline en/hosting_guide/site_creation.rst]: 4 steps i think

File: en/user_guide/websites_creation.rst [hunk 2]
```diff
@@ -1,73 +1,92 @@
 Website creation
 ================
 
-Open Orchestra is multisite and allows to create several websites from the same Back Office application.
+Open Orchestra is multisite allowing the creation of several websites from the same Back Office
+application.
+
+So, the first first thing to do to create a new site is to goi to the Back Office and fill the
```
Comment (itkg-rboessel) [inline en/user_guide/websites_creation.rst]: is to goi

File: en/user_guide/websites_creation.rst [hunk 3]
```diff
@@ -1,73 +1,92 @@
 Website creation
 ================
 
-Open Orchestra is multisite and allows to create several websites from the same Back Office application.
+Open Orchestra is multisite allowing the creation of several websites from the same Back Office
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: the previous formulation was better

File: en/user_guide/websites_creation.rst [hunk 4]
```diff
@@ -1,73 +1,92 @@
...
+A site may exist in different languages, and each of this versions could have minor differences, for
+instance a node could be published in English and not in French. Moreover, a site in English could be
+accessed from different domains. To make the managment of those problematics easier, Open Orchestra
+introduces the concept of site alias.
+Basically a site represents a set of site aliases sharing contents and pages, each site alias in a
+single language. Multiple site aliases can be configured with the same language, if they define
+different access points, eg domain + language prefix.
 
-For example :
+For example, the site http://demo.open-orchestra.com/ exists in english and french. It presents two
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: for instance

File: en/user_guide/websites_creation.rst [hunk 5]
```diff
@@ -1,73 +1,92 @@
...
-Each node can also have its own theme.
+The field theme is the default theme used by the website.
+This setting can be overriden node by node in order to have a specific theme on a special node.
 
 Sitemap generation (required)
 -----------------------------
 
-The two folling attributes are used for the sitemap generation :
+The two following settings are related to the sitemap.xml generation. That file help Search Engine
+Optimization. Those two settings are default values for all nodes of the site, but they can be
```
Comment (juchi) [inline en/user_guide/websites_creation.rst]: repetition of "the two settings"
"Those two settings are default values" => have default values

File: en/hosting_guide/site_creation.rst [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+* Dealing with rights
+* Contributing the home page
+* Creating an Apache virtual host
+* (Optionnal) Configure the 404 & 503 pages
+* (Optional) Generate the robots.txt
+* (Optional) Generate the sitemap.xml
+
+Declaring the site
+------------------
+The first thing to do, is to declare the site in Open Orchestra. To do this there is no need to open
```
Comment (juchi) [inline en/hosting_guide/site_creation.rst]: the comma should be removed
the second sentence brings few information, i would remove it and just leave :
"[...] declare the site in Open Orchestra by filling the website creation form"

File: en/hosting_guide/site_creation.rst [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+a file, you only have to log in the Back Office and fill the website creation form.
+
+Full details are available in the user guide, on `website creation page`_.
+
+Dealing with rights
+-------------------
+To be able to contribute to the site, rights have to be set. To do this, create a new group
```
Comment (juchi) [inline en/hosting_guide/site_creation.rst]: access rights ?

File: en/hosting_guide/site_creation.rst [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+including several roles related to this site (including ROLE_ACCESS_TREE_NODE) and assign it to the
+newly created site.
+
+Once this is done, you can add users to the group by editing them. On their next login, these users
```
Comment (juchi) [inline en/hosting_guide/site_creation.rst]: "login" is a username, "log in" is the action. Anyway, I think in this context we should say "their next connection"

File: en/hosting_guide/site_creation.rst [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+will be able to select that site with the site switcher. 
+
+.. image:: ../../images/site_switcher.png
+
+Further information on group contribution is available in the user guide, on `user page`_
+
+Contributing the home page
+--------------------------
+To get sure the site is accessible, you should have something to see. So it's a good idea to
+contribute a homepage.
```
Comment (juchi) [inline en/hosting_guide/site_creation.rst]: Is this really useful to say ?

File: en/hosting_guide/site_creation.rst [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+First thing is to select the newly created site on the site switcher. Then, under the Editorial part
+of the menu, the 'Pages' entry should look like this:
+
+.. image:: ../../images/new_site_tree.png
+
+Clicking on 'New page' will allows you to contribute the homepage of the site. You can go to the
```
Comment (juchi) [inline en/hosting_guide/site_creation.rst]: allow

File: en/hosting_guide/site_creation.rst [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+user guide, on the `node page`_ and `node configuration page`_ for full details about how to edit a
+node.
+
+Once contributed, publish the node (Ideally, in all the languages used by your site aliases).
+
+Creating an Apache virtual host
+-------------------------------
+Now, Open Orchestra knows about your site and a first page is contributed. The last thing to do is
+to get that page reachable. To do this you have to create an Apache virtual host to serve the
+content. Here is a simple example working:
```
Comment (juchi) [inline en/hosting_guide/site_creation.rst]: Here is a simple example:

File: en/hosting_guide/site_creation.rst [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+        CustomLog /absolute/path/to/logs/mysite.openorchestra.access.log combined
+    </VirtualHost>
+
+
+Note that the DocumentRoot must point to the web directory of Open Orchestra installation, not on
+the Open Orchestra root directory.
+
+Once Apache restarted, your new website should be accessible.
+
+* Note 1: If the domain name is a local one, don't forget to add it to your hosts file.
```
Comment (juchi) [inline en/hosting_guide/site_creation.rst]: that's kind of obvious IMHO

File: en/hosting_guide/site_creation.rst [hunk 13]
```diff
@@ -0,0 +1,106 @@
+Website creation
+================
+
+Creating a new site with Open Orchestra requires three steps. Additionnally, you can add some more
+steps to enhance the site features. Those steps are the following:
+
+* Declaring the website in Open Orchestra
+* Dealing with rights
+* Contributing the home page
+* Creating an Apache virtual host
```
Comment (juchi) [inline en/hosting_guide/site_creation.rst]: I wouldn't say "apache" as you don't want to limit yourself. This section can contain an apache example but should be server agnostic

Comment (ngilain) [inline en/hosting_guide/site_creation.rst]: Officially Open Orchestra requires an Apache server, but you're right, I'll try to be as transparent as possible

**Comment by ngilain** [comment]
@itkg-rboessel , @juchi : updates done

**Comment by juchi** [comment]
:+1:

File: en/hosting_guide/site_creation.rst [hunk 14]
```diff
@@ -0,0 +1,104 @@
...
+
+        ErrorLog /absolute/path/to/logs/mysite.openorchestra.error.log
+        CustomLog /absolute/path/to/logs/mysite.openorchestra.access.log combined
+    </VirtualHost>
+
+
+Note that the DocumentRoot must point to the web directory of Open Orchestra installation, not on
+the Open Orchestra root directory.
+
+Once Apache restarted, your new website should be accessible.
```
Comment (nicolasThal) [inline en/hosting_guide/site_creation.rst]: You don't need to restart apache, a simple reload will reload the configuration (if I have a good memory)

File: en/summary.rst [hunk 15]
```diff
@@ -109,3 +110,4 @@ Hosting guide
 .. _`Use a specific backbone view`: /en/developer_guide/specific_backbone_view.rst
 .. _`Add a special listener to a Content field`: /en/developer_guide/content_add_field_listener.rst
 .. _`Use the ajax pagination for the entities list`: /en/developer_guide/entity_list_ajax_pagination.rst
+.. _`Creating a new site` : /en/hosting_guide/site_creation.rst
```
Comment (nicolasThal) [inline en/summary.rst]: Could you respect the tree :) 
(we put the new line between line having the same size )

File: en/user_guide/websites_creation.rst [hunk 16]
```diff
@@ -3,71 +3,89 @@ Website creation
...
-The attribute theme is the default theme used by the website.
-Each node can also have its own theme.
+The field theme is the default theme used by the website.
+This setting can be overriden node by node in order to have a specific theme on a special node.
 
 Sitemap generation (required)
 -----------------------------
 
-The two folling attributes are used for the sitemap generation :
+The two following settings are related to the sitemap.xml generation. That file helps Search Engine
```
Comment (nicolasThal) [inline en/user_guide/websites_creation.rst]: I would not put the explanation on what the sitemap is created for at the same place that the rest of the sentence.

Would it be possible to put in a note just under ? or a link to what a sitemap is?

File: en/user_guide/websites_creation.rst [hunk 17]
```diff
@@ -3,71 +3,89 @@ Website creation
...
+This field lists all the blocks available to nodes contribution for this website.
+Only the selected blocks will be available.
+
 Default Theme (required)
 ------------------------
 
-The attribute theme is the default theme used by the website.
-Each node can also have its own theme.
+The field theme is the default theme used by the website.
+This setting can be overriden node by node in order to have a specific theme on a special node.
```
Comment (itkg-rboessel) [inline en/user_guide/websites_creation.rst]: overridden (idem for all other overriden)

File: en/user_guide/websites_creation.rst [hunk 18]
```diff
@@ -3,71 +3,89 @@ Website creation
...
+must be unique among all the sites.
+
 Aliases
 -------
+.. image:: ../../images/aliases.png
 
-Open Orchestra allows to create  several aliases for a site.
+A site may exist in different languages, and each of this versions could have minor differences, for
+instance a node could be published in English and not in French. Moreover, a site in English could be
+accessed from different domains. To make the managment of those problematics easier, Open Orchestra
```
Comment (itkg-rboessel) [inline en/user_guide/websites_creation.rst]: management

File: en/hosting_guide/site_creation.rst [hunk 19]
```diff
@@ -0,0 +1,104 @@
+Website creation
+================
+
+Creating a new site with Open Orchestra requires four steps. Additionnally, you can add some more
```
Comment (itkg-rboessel) [inline en/hosting_guide/site_creation.rst]: Additionally

File: en/hosting_guide/site_creation.rst [hunk 20]
```diff
@@ -0,0 +1,104 @@
...
+        CustomLog /absolute/path/to/logs/mysite.openorchestra.access.log combined
+    </VirtualHost>
+
+
+Note that the DocumentRoot must point to the web directory of Open Orchestra installation, not on
+the Open Orchestra root directory.
+
+Once Apache restarted, your new website should be accessible.
+
+* Note: According to your environnment, be sure to call the right front controller when testing the
```
Comment (itkg-rboessel) [inline en/hosting_guide/site_creation.rst]: environment

File: en/hosting_guide/site_creation.rst [hunk 21]
```diff
@@ -0,0 +1,104 @@
...
+access (eg app_dev.php, app_prod.php, etc ...).
+
+(Optionnal) Configure the 404 & 503 pages
```
Comment (itkg-rboessel) [inline en/hosting_guide/site_creation.rst]: optional

File: en/hosting_guide/site_creation.rst [hunk 22]
```diff
@@ -0,0 +1,104 @@
...
+================
+
+Creating a new site with Open Orchestra requires four steps. Additionnally, you can add some more
+steps to enhance the site features. Those steps are the following:
+
+* Declaring the website in Open Orchestra
+* Dealing with access rights
+* Contributing the home page
+* Configuring the webserver
+* (Optionnal) Configure the 404 & 503 pages
```
Comment (itkg-rboessel) [inline en/hosting_guide/site_creation.rst]: optional

File: en/hosting_guide/site_creation.rst [hunk 23]
```diff
@@ -0,0 +1,104 @@
...
+
+(Optionnal) Configure the 404 & 503 pages
+-----------------------------------------
+At this stage, your site is visible. But it's the minimum and you can add several optional features.
+For instance, you can customize the 404 and 503 pages to map them to your site design. The
+`documentation on error pages`_ shows how to do that.
+
+(Optional) Generate the robots.txt
+----------------------------------
+Additionnally, you can generate the robot.txt file according to the setting configured previously
```
Comment (itkg-rboessel) [inline en/hosting_guide/site_creation.rst]: Additionally

**Comment by itkg-rboessel** [comment]
:+1: for me after modifications

File: en/summary.rst (hunk 15 above)
Comment (ngilain) [inline en/summary.rst]: I can but what is the purpose ? It makes the links hard to maintain :s

**Comment by ngilain** [comment]
@nicolasThal : update done

**Comment by nicolasThal** [comment]
:+1: after a little squash

**Pull request merged**